classification/models/fishing_detection.py

```diff
@@ -0,0 +1,224 @@
+# Copyright 2017 Google Inc. and Skytruth Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+import argparse
+import json
+from .model import ModelBase
+from . import layers
+from classification import metadata
+from .objectives import (
+    FishingLocalizationObjectiveCrossEntropy, TrainNetInfo)
+from classification.feature_generation import fishing_feature_generation
+import logging
+import math
+import numpy as np
+import os
+
+import tensorflow as tf
+
+approx_means = [ 6.9139094e+00,  4.3506036e+00,  6.1346573e-01,  5.7922113e-01,
```

Comment:
Any chance of a comment here? I assume they are per-feature mean/stds for input normalisation? Also be nice to know how you estimated them.

Reply:
Will do.